synapse/storage/databases/main/search.py

```diff
@@ -604,7 +611,7 @@ async def search_rooms(
             search_query = search_term
             sql = """
             SELECT ts_rank_cd(vector, websearch_to_tsquery('english', ?)) as rank,
-            origin_server_ts, stream_ordering, room_id, event_id
+            room_id, event_id, origin_server_ts, stream_ordering
```

Comment:
is this just ensuring we use a consistent column order here, so that we can compare tuples from two such queries?

Reply:
Oh, I think it's ensuring consistency with the SQLite query?

Reply:
It is making the columns the same across SQLite and PostgreSQL